stan/math/prim/prob/von_mises_cdf.hpp

```diff
@@ -0,0 +1,124 @@
+#ifndef STAN_MATH_PRIM_PROB_VON_MISES_CDF_HPP
+#define STAN_MATH_PRIM_PROB_VON_MISES_CDF_HPP
+
+#include <stan/math/prim/fun/constants.hpp>
+#include <stan/math/prim/prob/normal_cdf.hpp>
+#include <stan/math/prim/fun/modified_bessel_first_kind.hpp>
+#include <cmath>
+
+namespace stan {
+namespace math {
+
+namespace internal {
+
+template <typename T_x, typename T_k, typename T_p>
+return_type_t<T_x, T_k, T_p> von_mises_cdf_series(const T_x& x, const T_k& k,
+                                                  const T_p& p) {
+  using stan::math::cos;
+  using stan::math::sin;
```

Comment:
You are already inside of the math namespace so these should be visible. You may actually want to do 
```suggestion
  using std::cos;
  using std::sin;
```

to make sure those are visible to the compiler

Reply:
We don't want the std ones though? We really do want the stan::math ones. Just leaving stan::math out is enough.